src/bitfield.rs

```diff
@@ -289,7 +289,7 @@ mod tests {
 		].iter().cloned().collect();
 
 		a.set_bit(0, 10).unwrap(); // prevote 1
-		a.set_bit(11, 10).unwrap(); // precommit 2
+		a.set_bit(3, 10).unwrap(); // precommit 2
```

Comment:
a comment here saying the `VoterSet` is sorting internally might be a good idea.